index.html

```diff
@@ -0,0 +1,139 @@
+      <a href="#electronic_instruments" class="nav-link">Electronic Instruments</a>
+    </nav>
+    <main id="main-doc">
+      <section class="main-section" id="main_types_of_instruments">
+        <header>Main Types of Instruments </header>
+        <article>
+          <code></code>
+          <p>
+            Musical instruments are special devices that produce various tunes and melodies, which combine to form music. In fact, any object or device that can create a harmonious sound can be regarded as a musical instrument.
+          </p>
+          <p>
+            The history of musical instruments dates back to as far as 67,000 years ago. The oldest object that is referred to as a musical instrument by some scholars is a flute.
+          </p>
+          <p>
+            The classification of musical instruments is a disciple in its own right. There have been many classifications of musical instruments to date. Most of them have tried to classify the musical instruments based on various factors, such as their effective range, size, role, and the composition of the material used.
+          </p>
+          <p>
+            But, the most widely accepted classification of musical instruments is the way they produce sound. The study of musical instruments is known as organology.
+          </p>
+          <p>
+            If you are interested in learning musical instruments, you must familiarize yourself with all of their types. Even if you are not interested in playing one, it is still exhilarating to explore different musical instruments.
+          </p>
+          <p>
+            It will help you to decide what kind of learning you wish to attain in the niche of music. So, here is the list of all the major classes of musical instruments.
+          </p>
+        </article>
+      </section>
+      <section class="main-section" id="wind_instruments">
+        <header>Wind Instruments</header>
+        <article>
+          <code></code>
+          <p>
+            In this type of musical instrument, all you need to do is blow into the musical instrument by following a defined order to produce a particular kind of sound. These instruments essentially work on the principles of sound waves, resonance, and harmonics. We can control the frequencies, and acoustics can be set and altered by changing how we blow into the instrument and what order we follow.
+          </p>
+          <p>
+            There is a wide variety of wind instruments that can be seen around the world. Some of the highly renowned wind instruments are:
+          </p>
+          <ul>
+            <li>Flute</li>
+            <li>Oboe</li>
+            <li>Clarinet</li>
+            <li>Bassoon</li>
+            <li>Saxophone</li>
+          </ul>
+        </article>
+      </section>
+      <section class="main-section" id="string_instruments">
+        <header>String Instruments</header>
+        <article>
+          <code></code>
```

Comment:
Could you please consider removing the empty tags

Reply:
- [ ] Kindly check my comments on line `47` above on the `why` implementation 👍